Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift

```diff
@@ -78,6 +83,12 @@ final class SubscriptionsViewController: BaseViewController {
         tableView.reloadData()
         registerKeyboardHandlers(tableView)
     }
+
+    @IBAction func clearSearch(_ sender: UIButton) {
+        textFieldSearch.text = ""
+        searchBy()
```

Comment:
Is this call needed? I believe it's not, because the textField delegate will be called.